sdcm/cluster_k8s/__init__.py

```diff
@@ -108,8 +109,26 @@ def deploy_cert_manager(self) -> None:
                                namespace="cert-manager"))
         time.sleep(10)
         self.kubectl("wait --timeout=10m --all --for=condition=Ready pod", namespace="cert-manager")
+        wait_for(
+            self.check_if_cert_manager_fully_functional,
+            text='Waiting for cert-manager to become fully operational',
+            timeout=10 * 60,
+            step=10,
+            throw_exc=True)
         self.start_cert_manager_journal_thread()
 
+    def check_if_cert_manager_fully_functional(self):
+        # It is not possible to say with 100% certainty that cert-manager become fully operational
+        # In this function we use dirty way to estimate that by deploying scylla-operator
+        # which invokes cert-manager hooks and making sure that next step after cert-manager deploying,
+        # which is scylla-operator deploying, won't fail.
+        try:
+            self.apply_file(SCYLLA_OPERATOR_CONFIG)
+            return True
+        finally:
+            self.kubectl('delete namespace scylla-operator', ignore_status=True)
+        return False
+
```

Comment:
It's too dirty :) Can we use an example from https://cert-manager.io/docs/installation/kubernetes/#verifying-the-installation ?

Reply:
fixed